src/main/java/edu/ucsb/cs56/mapache_search/GoogleSearchService.java

```diff
@@ -55,10 +55,12 @@ public String getJSON(String query, String apiKey) {
 
         HttpEntity<String> entity = new HttpEntity<>("body", headers);
 
+        int startIndex = ((params.getPage() - 1) * 10) + 1;
+
         String uri = "https://www.googleapis.com/customsearch/v1";
-        String params = String.format("?key=%s&cx=%s&q=%s&alt=json",
-            apiKey, searchId, query);
-        String url = uri + params;
+        String queryParams = String.format("?key=%s&cx=%s&q=%s&alt=json&start=%d",
+            apiKey, searchId, params.getQuery(), startIndex);
+        String url = uri + queryParams;
         logger.info("url=" + url);
```

Comment:
This could use `resttemplate` as done in #67

Reply:
We made an issue for this to be looked at in the future

Reply:
@pconrad #67 actually fixes this already, this branch just hasn't been rebased onto master after #67 was pulled in.